### DISCUSSION TITLE
Light Mode

### DISCUSSION THREAD

**Discussion by taylor1021** [Zibbp/ganymede#397] [Q&A] [answered]
I've been using the Light Theme, but upon upgrading from v1.3.0 to 2.1.0, the frontend switched to the Dark Theme. I've looked through the UI to find a toggle to switch back to the Light Theme, but I can't find it anywhere. How can the UI be switched to Light?

**Comment by Zibbp** [comment, marked as answer]
I removed that as I figured most people are using dark mode. I've re-implemented the dark/light theme button at the top right of the navbar in [v2.2.0](https://github.com/Zibbp/ganymede/releases/tag/v2.2.0) 
![image](https://github.com/Zibbp/ganymede/assets/21207065/2b25c3ea-ca6c-4a49-8484-2e9c3b7fb6c7)

**Reply by taylor1021** [reply]
Thanks for restoring the light mode; I appreciate it!
